metagraph/src/cli/config/config.cpp

```diff
@@ -393,6 +393,8 @@ Config::Config(int argc, char *argv[]) {
             arity_brwt = atoi(get_value(i++));
         } else if (!strcmp(argv[i], "--relax-arity")) {
             relax_arity_brwt = atoi(get_value(i++));
+        } else if (!strcmp(argv[i], "--row-sparse-disk-buff-size")) {
```

Comment:
please add appropriate help message lines for this new flag so it show up when one types `--help`

Reply:
Done